app/assets/javascripts/discourse/models/composer.js.es6

```diff
@@ -195,6 +195,11 @@ const Composer = RestModel.extend({
     return canEditTopicFeaturedLink ? !this.siteSettings.topic_featured_link_onebox : true;
   },
 
+  @computed("topic")
+  draftTitle(topic) {
```

Comment:
That is also wrong though :)

If you have to emoji the title, put the property in the controller instead of the model. You can leave the icon in the template.